public/index.html

```diff
@@ -144,7 +144,9 @@
               store.setHistory(annotationHistory)
             }
           })
-
+          ls.on("updateAnnotation", (_,annotation) => {
```

Comment:
Actually that's a good thing to leave on the demo page!
But please, lint it — `_, `, `})`, semicolons